### DISCUSSION TITLE
API token generation [Jan 28]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5037] [Discord Threads]
> 🙋‍♂️ **Saurav 😴**

**Comment by o1lab** [comment]
> 👨‍💻 **pranavxc**  
#### In api snippet it showing auth token and it's valid for certain amount of time. What you have to do is create a token from account settings and pass it in header as `xc-token`.  
#  
> 👨‍💻 **pranavxc**  
####   
![image0](https://cdn.discordapp.com/attachments/1068822062304600084/1068827158304276531/Screenshot_2023-01-28_at_3.06.04_PM.png)  
![image1](https://cdn.discordapp.com/attachments/1068822062304600084/1068827158576910376/Screenshot_2023-01-28_at_3.06.19_PM.png)  
#  
> 🙋‍♂️ **Saurav 😴**  
#### right, thanks  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1068822062304600084)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
